src/script/components/MessagesList/MessageList.styles.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {CSSObject} from '@emotion/react';
+
+export const jumpToLastMessageButtonStyles: CSSObject = {
+  position: 'absolute',
+  right: '10px',
+  height: '40px',
+  borderRadius: '100%',
+  bottom: '56px',
+
+  '@media (max-width: 768px)': {
+    bottom: '100px',
+  },
```

Comment:
@V-Gira I see this value being copied a few times in the app. Not a topic for now, but we should have a single place where the different breakpoints are defined so we avoid hardcoding breakpoints like this (and when possible those breakpoints should not be used in favor of just relying on the natural DOM flow).



I did try to use the DOM flow to have the button placing naturally, but turns out it's not really possible to have a button that is fixed in a scrolling container sadly. 



Let's try to optimize this in a followup PR

Reply:
I also see that we have lots of hardcoded breakpoints in the code, and they are different. We should define key breakpoints, and then each component should use them and introduce new ones as `theme.breakpoints.tablet - COMPONENT_CUSTOM_WIDTH_CONSTANT` etc.